project3/styles.css

```
@@ -0,0 +1,361 @@
+/* Font & Color Palette */
+
+:root {
+    --color-red: #c61a3a;
+    --color-green: #71f79f;
+    --color-offblack: #1E2019;
+    --color-grey: #d1d5de;
+    --color-offwhite: #fafafa;
+    --color-blue: #163346;
+
+    --font-body: 'BenchNine', sans-serif;
+    /* Weights: 300, 400, 700 */
+
+    --font-heading: 'Teko', sans-serif;
+    /* Weights: 500, 600, 700 */
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+/* Utility Classes */
+
+.button-reset {
+    background: transparent;
+    border: 0;
+    cursor: pointer;
+    font-family: inherit;
+    font-size: inherit;
+    font-weight: inherit;
+    padding: 0;
+    text-align: inherit;
+}
+
+.button-reset[disabled] {
+    cursor: default;
+}
+
+/* Global Settings */
+
+body {
+    background-color: var(--color-blue);
+    color: white;
+    font-family: var(--font-body);
+    font-size: 22px;
+    margin: 0;
+}
+
+button[type="submit"] {
+    background-color: var(--color-green);
+    color: black;
+    clear: both;
+    display: block;
+    font-family: var(--font-heading);
+    font-size: 32px;
+    height: 87px;
+    margin: 120px auto 80px auto;
+    text-align: center;
+    text-transform: uppercase;
+    transition: all 300ms;
+    width: 205px;
+}
+
+button:hover {
```

Since you were specific with your initial CSS, I'd keep the same selector for your hover styles: 

```suggestion
button[type="submit"]:hover {
```